### PR TITLE
added Puppeteer.executablePath tests

### LINKED ISSUE #78: Implement Puppeteer.executablePath test
https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/test/test.js#L241

### PR DESCRIPTION
**Pull request by Meir017**
closes #78

### CODE REVIEW

**Comment by kblok** [comment]
Well not AppVeyor is working :joy:

**Comment by Meir017** [comment]
It's failing because:
image

File: lib/PuppeteerSharp/Downloader.cs [hunk 1]
```diff
@@ -21,6 +21,8 @@ public class Downloader
         };
         private string _downloadHost;
 
+        public const int DefaultRevision = 526987;
```
Comment (kblok) [inline lib/PuppeteerSharp/Downloader.cs]: I can buy this now. But I created an issue to make it configurable in v1.0 https://github.com/kblok/puppeteer-sharp/issues/84

File: lib/PuppeteerSharp/Puppeteer.cs [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 {
     public class Puppeteer
     {
         public static string[] DefaultArgs => Launcher.DefaultArgs;
 
+        public static string GetExecutablePath()
+        {
+            var downloader = Downloader.CreateDefault();
+            var revisionInfo = downloader.RevisionInfo(Downloader.CurrentPlatform, Downloader.DefaultRevision);
+            return revisionInfo.ExecutablePath;
```
Comment (kblok) [inline lib/PuppeteerSharp/Puppeteer.cs]: This code is in the Launcher class on Puppeteer. Let's honor that.

https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/Launcher.js#L188

Comment (Meir017) [reply lib/PuppeteerSharp/Puppeteer.cs]: should that method be static?

Comment (kblok) [reply lib/PuppeteerSharp/Puppeteer.cs]: Yes. Let's follow Puppetter API, it's static there.

File: lib/PuppeteerSharp/Launcher.cs [hunk 3]
```diff
@@ -136,6 +136,12 @@ public class Launcher
             }
 
         }
+        internal string GetExecutablePath()
```
Comment (kblok) [inline lib/PuppeteerSharp/Launcher.cs]: Can we add one line between the bracket on line 138 and this new function?

File: lib/PuppeteerSharp.Tests/Puppeteer/ExecutablePathTests.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿using System.IO;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Puppeteer
+{
+    public class ExecutablePathTests
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/Puppeteer/ExecutablePathTests.cs]: You need to make this class inherit from `PuppeteerBaseTest` and add the attribute `[Collection("PuppeteerLoaderFixture collection")]`

The problem we have here is that, unluckily, this is the first test running, so we need to download Chromium first, and `PuppeteerBaseTest` does that job for us.

Comment (kblok) [reply lib/PuppeteerSharp.Tests/Puppeteer/ExecutablePathTests.cs]: @Meir017 do you think you will be able to implement this by tomorrow? 
If not I can merge it and implement it in another PR.

**Pull request merged**